### ISSUE TITLE
loading checkpoint

### ISSUE THREAD

**Issue by happisky-lyt** [mbzuai-oryx/GeoChat#24]
I encountered the following issue when loading the checkpoint
![issue](https://github.com/mbzuai-oryx/GeoChat/assets/128709110/b4d06f5b-abe4-477b-876d-832c9e0aaed1)
can u help me to solve it?

**Comment by Hoteryoung** [comment]
@happisky-lyt
Hi, sorry to bother you, but I had the same problem.
Have you solved it?

**Comment by happisky-lyt** [comment]
I first ran the demo on my own pc, and I set line 103 of `geochat.model.builder.py` as followed:

`model = GeoChatLlamaForCausalLM.from_pretrained(model_path, offload_folder="offload", low_cpu_mem_usage=True, **kwargs)`

But when doing inference there is an OOM error, so I uploaded the code to the server and I got the error I mentioned before.
i believe the error was caused by setting `offload_folder="offload"`, so I directly download the repository in my server and done inference successfully.

hope this helps you.
